Automattic/wp-calypso
# Jetpack connect: Separate from signup module

This PR moves `jetpack-connect` out of the signup module and into its own module (finally 🎉 ).

The changes are actually pretty small and straightforward after all the prepwork tracked in #12991.

To test:
* Use this branch
* Use Jetpack connect! Relevant entry points include:
  * `calypso.localhost:3000/jetpack/connect`
  * `calypso.localhost:3000/jetpack/connect/?url=:site`
  * `calypso.localhost:3000/jetpack/connect/store`
  * `calypso.localhost:3000/jetpack/connect/personal`
  * `calypso.localhost:3000/jetpack/connect/premium`
  * `calypso.localhost:3000/jetpack/connect/professional`
  * See [this file](https://github.com/Automattic/wp-calypso/blob/00e998ec01144bcc4ba1596c121f81d7f48c94a1/client/jetpack-connect/index.js) for a more complete list.
* Please pay special attention to the module level changes and raise _any_ questions as I'm not very familiar with that part of the codebase.

Fixes #14321, Supersedes #12991
